integreat_cms/cms/constants/roles.py

```diff
@@ -87,6 +87,9 @@
 AUTHOR_PERMISSIONS: Final[list[str]] = EVENT_MANAGER_PERMISSIONS + [
     "change_page",
     "view_page",
+    "change_contact",
+    "delete_contact",
+    "view_contact",
```

Comment:
In the current setting authors, editors and managers can access contacts but none of the staff users, and we need to restrict access to this new feature to CMS + Service Team or CMS + the product owner until everything is finished and we finalize the permission settings in #2958

Reply:
I think there was some misunderstanding between David and me beforehand that caused this. However, I think this is resolved now, and therefore I changed the permissions for only service-team and cms-team to be able to access the contact model